aspnetcore/blazor/hosting-models.md

```diff
@@ -291,16 +387,22 @@ Blazor WebAssembly apps can run offline, which is particularly useful when clien
 
 ### Static site hosting
 
-Static site hosting is possible with Blazor WebAssembly apps because they're downloaded to clients as a set of static files. Blazor WebAssembly apps don't require a server to execute server-side code in order to download and run. Blazor WebAssembly apps can be delivered via a [Content Delivery Network (CDN)](https://developer.mozilla.org/docs/Glossary/CDN) (for example, [Azure CDN](https://azure.microsoft.com/services/cdn/)).
+Static site hosting is possible with static Blazor WebAssembly apps because they're downloaded to clients as a set of static files. Static Blazor WebAssembly apps don't require a server to execute server-side code in order to download and run and can be delivered via a [Content Delivery Network (CDN)](https://developer.mozilla.org/docs/Glossary/CDN) (for example, [Azure CDN](https://azure.microsoft.com/services/cdn/)).
```

Comment:
```suggestion
Static site hosting is possible with standalone Blazor WebAssembly apps because they're downloaded to clients as a set of static files. Static Blazor WebAssembly apps don't require a server to execute server-side code in order to download and run and can be delivered via a [Content Delivery Network (CDN)](https://developer.mozilla.org/docs/Glossary/CDN) (for example, [Azure CDN](https://azure.microsoft.com/services/cdn/)).
```